src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,35 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.14.0
+
+[DATE]
+
+This version is compatible with Kubernetes versions 1.24 to 1.27.
+
+### Deprecation Notice
+
+- As of this versions deploying helm charts through the Okteto UI has been deprecated.
+- Private Registry credentials defined in helm has been deprecated. Private Registry Credentials are now supported natively by Okteto and credentials defined in helm will be automatically migrated. More info [here](ADD_LINK)
+- Support for Kubernetes [1.24](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md) has been deprecated and will be removed in the next release.
+
+### Features
+
+- Expose prometheus node metrics data in `/metrics`. More info [here](ADD_LINK).
+- Native support for Private Registry Credentials. More info [here](ADD_LINK).
```

Comment:
This could be the link for this: https://www.okteto.com/docs/1.14/administration/dashboard/#registry-credentials

(We would have to remove `1.14` from the path so we always way to current version)